### PR TITLE
Address some JavaDoc errors/warns in some classes

### PR DESCRIPTION
**Pull request by thc202**
Address JavaDoc errors and warnings mostly in API related classes.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/api/API.java [hunk 1]
```diff
@@ -421,12 +421,12 @@ public boolean handleApiRequest (HttpRequestHeader requestHeader, HttpInputStrea
 	
 	/**
 	 * Returns a URI for the specified parameters. The API key will be added if required
-	 * @param format
-	 * @param prefix
-	 * @param type
-	 * @param name
+	 * @param format the API format
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/api/API.java]: Response format?

File: src/org/zaproxy/zap/extension/api/API.java [hunk 2]
```diff
@@ -421,12 +421,12 @@ public boolean handleApiRequest (HttpRequestHeader requestHeader, HttpInputStrea
...
+	 * @param prefix the prefix of the API implementor
+	 * @param type the API type
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/api/API.java]: View, action, other etc? Should we clarif you or provide examples?

File: src/org/zaproxy/zap/extension/api/ApiDynamicActionImplementor.java [hunk 3]
```diff
@@ -47,6 +45,13 @@ public ApiResponse buildParamsDescription() {
 		return configParams;
 	}
 
+	/**
+	 * Builds a {@code ApiResponseSet} with the given parameter name and whether or not is mandatory.
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/api/ApiDynamicActionImplementor.java]: "..or not it is...". Looks like there's a double space.

File: src/org/zaproxy/zap/extension/api/API.java (hunk 1 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/api/API.java]: Yeah.

File: src/org/zaproxy/zap/extension/api/API.java (hunk 2 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/api/API.java]: That should be done by the enum.
Will tweak the text to "the request type".

File: src/org/zaproxy/zap/extension/api/ApiDynamicActionImplementor.java (hunk 3 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/api/ApiDynamicActionImplementor.java]: Noted.

File: src/org/zaproxy/zap/extension/api/API.java (hunk 1 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/api/API.java]: So let's call it "api response format"?

File: src/org/zaproxy/zap/extension/api/API.java (hunk 2 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/api/API.java]: K

File: src/org/zaproxy/zap/extension/api/API.java (hunk 1 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/api/API.java]: Something like that, yes.

Comment (thc202) [inline src/org/zaproxy/zap/extension/api/API.java]: Went with "the format of the API response".

**Comment by thc202** [comment]
Changes done, thanks.

**Comment by kingthorin** [comment]
Looks good, thanks.

**Pull request merged**